RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py

```diff
@@ -128,9 +128,9 @@
 initialStepChi2EstPreSplitting = RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi.Chi2ChargeMeasurementEstimator.clone(
     ComponentName = cms.string('initialStepChi2EstPreSplitting'),
     nSigma = cms.double(3.0),
-    MaxChi2 = cms.double(30.0),
+    MaxChi2 = cms.double(16.0),
```

Comment:
updates in this file look more than just technical optimization numerics.

Please add some MTV or hiteff plots for vfp30 case.